GuildedRoseLLC/Items/RemoteItemRepository.swift

```diff
@@ -2,12 +2,21 @@ import UIKit
 
 public class RemoteItemRepository: ItemRepository {
     
-    public init() {}
+    public static var stagingAPI: String = "https://floating-spire-59497.herokuapp.com/api/v1/items"
+    public var url: String
+    
+    public init() {
+        url = RemoteItemRepository.stagingAPI
```

Comment:
Comment, no changes needed: If I understand this correctly, you are setting the url to the staging API url by default, but you have the option to change it via `setURL`, correct? I think this is ok for now. Just note that usually, details like URLs, keys, usernames, passwords, etc. are often stored in config files - sometimes these files are even encrypted. This setup might make it difficult to deploy one instance of the app to staging and one instance of the app to a production environment

Reply:
+1 to this and if your configured URLs are not malformed I would expect you to be able to remove the `guard let requestUrl = ...`.